### ISSUE TITLE
Confused about the design concept of Explicit quantization Q/DQ node in pytorh_quantizaiton toolkit 

### ISSUE THREAD

**Issue by Mavericky-j** [NVIDIA/TensorRT#1949]
Description
I am very confused about the design concept of the Q/DQ node during QAT. now that TensorRT has the implementation of int8 conv op,

why we cannot quantize weights and activations to int8 then use int8 conv op directly during explicit quantization?

and why we must use DQ to restore precision? If we act like this, take number 3 as example, is it like that we don’t use integer 3 but use float 3.0 instead? What’s the meaning of DQ node?

And one more question，why the int8 implementation is faster than fp16, is it because that the int8 has faster engine?

I would very appreciate if someone can give me some explanations!

**Comment by ttyio** [comment]
Hello @Mavericky-j 
My understanding for Q/DQ. The traditional PTQ issue is that the INT8 loss is not seem during training. With the inserted Q/DQ (aka, fake quant) nodes, now the quantization loss is also seems during training, DL framework would use the Q/DQ pair to quant the float tensor to int8 tensor, while the computation is still in FP32/FP16. By doing this we can minimalize the loss by fine-tune the weights. 
After export the ONNX, we still keep the Q/DQ, so that the I/O tensor precision is that same compare to training time and inference time. TensorRT also make the computation done in INT8 to accelerate your deployment.

For why INT8 is faster than FP16 , yes INT8 tensorcore has higher TFLOPS than FP16 (e.g, check A100 spec in https://www.nvidia.com/en-us/data-center/a100/), also INT8 has lower memory traffic requirement, better power efficiency. More details can we seen in GTC session in https://www.nvidia.com/en-us/on-demand/session/gtcspring21-s31653/

Thanks

**Comment by nvpohanh** [comment]
closing this issue for now due to >14 days of no response. Please feel free to reopen if the issue still exists. Thanks
